src/main/java/com/artipie/asto/rx/RxCopy.java

```diff
@@ -67,6 +66,15 @@ public RxCopy(final RxStorage from, final List<Key> keys) {
      * @return The completion signal.
      */
     public Completable copy(final RxStorage to) {
-        return Completable.error(new IllegalStateException("not implemented"));
+        return Completable.merge(
```

Comment:
@Sammers21 I think we need to handle the parallelness of copy, starting all downloads simultaneously may cause huge IO loads or OOM errors. E.g. I'm going to use this class in `rpm-adapter` which deals with thousands of storage objects, starting 5000 downloads at the same time is not a good idea. Maybe you can use IO scheduler here.